google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -31,6 +31,9 @@ appengine.flex.dockerfile.location.label=Dockerfile path\:
 appengine.flex.config.destination.folder.label=Choose destination folder\:
 appengine.flex.config.choose.destination.folder.window.title=Choose destination folder
 appengine.flex.config.destination.chooser.title=Choose Generated Configuration Destination Folder
+appengine.flex.config.destination.create.dir=Directory does not exist. Create it too now?
```

Comment:
Sorry for nitpicking, but "Create it too now" is a little awkward. Maybe drop the "too"?